GenderPayGap.WebUI/Views/Viewing/EmployerDetails/Quartiles/QuartilesBarChart.cshtml

```diff
@@ -14,10 +14,10 @@
 
     <div class="govuk-chart-horz__foot">
         <div class="govuk-chart-horz__text--left">
-            <span class="bold-small">@(Model.FemaleFormatted)% <span class="visually-hidden"> of the @Model.Title.ToLower() are women</span></span>
+            <span class="bold-small">@(Model.FemaleFormatted)% <span class="visually-hidden"> &nbsp of the @Model.Title.ToLower() are women</span></span>

```

Comment:
Why are we adding non-breaking spaces here? I also think this isn't right, they should have a `;` at the end.

Reply:
![image](https://user-images.githubusercontent.com/62190777/185180282-104e59e8-5e03-4944-bbcd-fbb0db3d8eee.png)
With and without the non-breaking space, interestingly it doesn't care about the ; but I will add it.

Reply:
Is that two spaces in the one with the nbsp? Probably doesn't matter since it will be read out. Have you tried actually listening to it?